Mathlib/FieldTheory/Galois/Basic.lean

```diff
@@ -254,6 +254,13 @@ theorem fixingSubgroup_fixedField [FiniteDimensional F E] : fixingSubgroup (fixe
   refine (algEquivEquivAlgHom (fixedField H) E).toEquiv.symm.trans ?_
   exact (fixingSubgroupEquiv (fixedField H)).toEquiv.symm
 
+/--
+A subgroup is isomorphic to the Galois group of its fixed field.
+-/
+def subgroupEquivAlgEquiv [FiniteDimensional F E] (H : Subgroup (E ≃ₐ[F] E)) :
```

Comment:
The two subgroups are equal, right? For `Subgroup` this a well behaved notion.

Reply:
Ah, in the second one there is `IntermediateField...`. You're right, sorry.

Reply:
No problem. Thanks for the review!